src/forms/forms.service.ts

```diff
@@ -37,16 +41,23 @@ export const formSelect = {
             },
         },
     },
-};
+} as Prisma.FormSelect;
 
 @Injectable()
 export class FormsService {
     constructor(private prisma: PrismaService) {}
 
-    getAllForms() {
-        return this.prisma.form.findMany({
+    async getAllForms() {
+        const data = await this.prisma.form.findMany({
             select: formSelect,
         });
+
+        const dataToReturn = data.map(({ questions, ...i }) => ({
+            ...i,
```

Comment:
Maybe add a descriptive name to `i` in `...i`. It's hard read what it is.

Reply:
Hi Tim, thank you for the feedback! That's true will modify it as well.